environment.yml

```diff
@@ -17,8 +16,8 @@ dependencies:
   - s3fs
   - universal_pathlib <0.2.0
   - boto3
-  - deltalake=0.15.3
+  - deltalake >=0.18
   # - dask-deltatable
   - pip
   - pip:
-    - git+https://github.com/fjetter/dask-deltatable.git@dask_expr
+    - git+https://github.com/dask-contrib/dask-deltatable.git@7706c22aa07f978f481ca4996dce8ca0121c812e
```

Comment:
Looks like `dask-deltatable==0.3.3` should fix the issue you ran into? https://github.com/dask-contrib/dask-deltatable/issues/81

Reply:
yup that works!